org.librarysimplified.http.refresh_token/src/main/java/org/librarysimplified/http/refresh_token/internal/LSSimplifiedRefreshTokenJSON.kt

```diff
@@ -0,0 +1,23 @@
+package org.librarysimplified.http.refresh_token.internal
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import java.io.InputStream
+
+/**
+ * Functions to deserialize an access token.
+ */
+
+object LSSimplifiedRefreshTokenJSON {
+  fun deserializeFromStream(
+    stream: InputStream?
```

Comment:
Why is this nullable?

Reply:
I was passing the inputstream directly from the response to there, but I guess I can check its nullability before that!